### PR TITLE
Convert afM, afm, aft commands to rzshell

### PR DESCRIPTION
**Pull request by p371jah**
**Your checklist for this pull request**
- [ ] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [ ] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

...

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

File: librz/core/tui/vmenus.c [hunk 1]
```diff
@@ -3285,7 +3285,7 @@ RZ_API void rz_core_visual_define(RzCore *core, const char *args, int distance)
 		}
 		break;
 	case 'j':
-		rz_core_cmdf(core, "afm $$+$F @0x%08" PFMT64x, off);
+		rz_core_cmdf(core, "afm $$+$F @ 0x%08" PFMT64x, off);
```
Comment (XVilka) [inline librz/core/tui/vmenus.c]: Please convert this to use the API, if possible.

**Review by XVilka — CHANGES_REQUESTED** [review]

File: librz/core/tui/vmenus.c (hunk 1 above)
Comment (p371jah) [reply librz/core/tui/vmenus.c]: Idk what `$$+$F` means

Comment (XVilka) [reply librz/core/tui/vmenus.c]: See the output of `?$?` - `$$` is the current seek, `$F` is the function beginning (same as `$FB`), at least according to the help.

Comment (p371jah) [reply librz/core/tui/vmenus.c]: Ok seems it is equal to (`core->offset` + `addr of function in core->offset`). Super weird.

**Review by XVilka — APPROVED** [review]

**Review by wargio — APPROVED** [review]

File: librz/core/tui/vmenus.c (hunk 1 above)
Comment (p371jah) [reply librz/core/tui/vmenus.c]: I guess it should be `$F+$FS`.

**Pull request merged**